aioseop_class.php

```diff
@@ -4717,7 +4717,8 @@ function admin_menu() {
 			$menu_name, $menu_name, apply_filters( 'manage_aiosp', 'aiosp_manage_seo' ), $file, array(
 				$this,
 				'display_settings_page',
-			)
+			),
+			aioseop_get_menu_icon()
```

Comment:
Was it possible to use a URL?

This could also use the class name of font icon; which would be more a future issue.

Reply:
As far as SVG files go, it has to be a base64-encoded SVG using a data URI. However, using Dashicons here is possible: 
> "Pass the name of a Dashicons helper class to use a font icon, e.g. 'dashicons-chart-pie'. "
Feel free to open an issue for that.